osu.Game.Rulesets.Catch/Mods/CatchModNoScope.cs

```diff
@@ -17,11 +15,6 @@ public class CatchModNoScope : ModNoScope, IUpdatableByPlayfield
     {
         public override LocalisableString Description => "Where's the catcher?";
 
-        [SettingSource(
-            "Hidden at combo",
-            "The combo count at which the catcher becomes completely hidden",
```

Comment:
Meant to leave a note, but the tooltip here is less contextually correct with this PR. Maybe fine, needs localisation in the future anyway, we can reassess at that point.